extensions/security/audittrail/applib/src/main/java/org/apache/causeway/extensions/audittrail/applib/dom/AuditTrailEntryRepository.java

```diff
@@ -38,6 +39,8 @@ public interface AuditTrailEntryRepository {
 
     AuditTrailEntry createFor(final EntityPropertyChange change);
 
+    Can<AuditTrailEntry> createForBulk(final Can<EntityPropertyChange> entityPropertyChanges);
```

Comment:
should be `default` for backward compatibliity

Reply:
Fixed